src/shared/components/ProfilePage/Stats/index.jsx

```diff
@@ -0,0 +1,225 @@
+/**
+ * Stats Page.  Displays the stats of a TopCoder member.
+ */
+/* eslint-env browser */
+import _ from 'lodash';
+import React from 'react';
+import PT from 'prop-types';
+import { Link } from 'react-router-dom';
+import { getRatingColor } from 'utils/tc';
+import Th from 'assets/images/th.svg';
+import { shouldShowGraph, getHistory, getSubTrackStats, getSummary, getDetails } from 'utils/memberStats';
+import DistributionGraph from './DistributionGraph';
+import HistoryGraph from './HistoryGraph';
+import styles from './styles.scss';
+import StatsModal from './StatsModal';
+import SRMStats from './SRMStats';
+
+
+class ProfileStats extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeGraph: 'history',
+      showModal: false,
+      isMobile: window.innerWidth < 768,
+    };
```

Comment:
Local state :(
Local state to handle viewport resizing, instead of SCSS media selectors :(((